subtests/docker_cli/run_volumes/run_volumes.py

```diff
@@ -42,7 +42,12 @@
 
 
 class run_volumes(SubSubtestCaller):
-    config_section = 'docker_cli/run_volumes'
+
+    def run_once(self):
+        ptc = self.config['pretest_cmd'].strip()
+        if ptc != '' and ptc is not None:
+            utils.run(ptc, timeout=self.config['docker_timeout'])
+        super(run_volumes, self).run_once()
```

Comment:
In every other instance I can find, super() is called before anything else. Is there a reason why ptc runs first here? If so, should that be documented?